### PR TITLE
Add: face emotion recognition application

### PR DESCRIPTION
**Pull request by 2013-11390**
I think folder path for this application could be changed. 
need enhancement for detecting side face and more accuracy for recognizing emotion

### CODE REVIEW

**Comment by 2013-11390** [comment]
Fix codacy issue except method docstring. I give example of this application with not train because I think it is enough to use this application by using pretrained weights, not train from scratch

File: example/face_emotion_recognition.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+"""
+You can train face emotions by yourself but I recommend you to use pretrained
+weights
+
+Download pretrained weights from the url below.
+https://github.com/omnis-labs-company/omnis/releases/download/
+0.0.8/face_emotion.h5
```
Comment (ik1ne) [inline example/face_emotion_recognition.py]: You shouldn't split http(s) links into multiline: pylintrc has exception for those type of lines.

Comment (2013-11390) [reply example/face_emotion_recognition.py]: I change the code please check it again please

**Comment by ik1ne** [comment]
Reviewed, since everything except for missing docstring looks fine I will squash merge this PR.

**Comment by 2013-11390** [comment]
Resolve #325

**Pull request merged**